### PR TITLE
Remove unnecessary validation in DREF Ops Update

### PR DESCRIPTION
**Pull request by frozenhelium**

### CODE REVIEW

**Comment by szabozoltan69** [comment]
Great one! I have had to found it when I fixed the "blank=True" condition for field-report-id. Sorry...

**Comment by szabozoltan69** [comment]
There are a lot of DREF-s which has no field report attached. So it's a de facto change.

File: dref/serializers.py [hunk 1]
```diff
@@ -440,13 +440,12 @@ def validate(self, data):
                         % (dref_operational_update.id, dref_operational_update.operational_update_number)
                     )
                 )
-        if self.instance and dref and dref.field_report is None:
-            raise serializers.ValidationError('Can\'t edit Operational Update for which dref field report is empty.')
```
[the remark this answers is not in the document]
Comment (szabozoltan69) [reply dref/serializers.py:444]: There are a lot of DREF-s which has no field report attached. So it's a de facto change.

[the remark this answers is not in the document]
Comment (frozenhelium) [reply dref/serializers.py:444]: Field Report is not a mandatory field now!

**Review by szabozoltan69 — APPROVED** [review]

**Pull request merged**